ckanext/agsview/public/js/ags_common.js

```diff
@@ -71,6 +72,14 @@
     var keys = Object.keys(properties);
     return '<div>' + keys.map(function (key) {
       var value = properties[key];
+      if (date_fields.indexOf(key) > -1) {
+        date_value = new Date(value);
+        value = date_value.toLocaleString('default', {
```

Comment:
Another bug with IE to do with using default as the locale. Seems like using `undefined` should work.